### ISSUE TITLE
When connecting my audio receiver to my 6950 XT via HDMI it disables OpenGL

### ISSUE THREAD

**Issue by Taldren76** [GPUOpen-Drivers/AMD-Gfx-Drivers#33]
### Discussed in https://github.com/GPUOpen-Drivers/AMD-Gfx-Drivers/discussions/32

<div type='discussions-op-text'>

<sup>Originally posted by **Taldren76** November 18, 2024</sup>
I use my Sony STR-DH790 Receiver for audio on my PC. If I plug my receiver in my  into my RX 6950 XT via HDMI it disables OpenGL and anything that is dependent on OpenGL, such as DaVinci Resolve Studio, will not function. I tried using DP to HDMI adapters in each  of the DP slots in addition to using the cards dedicated HDMI slot with the same issue. 

Driver version is 24.10.1</div>

**Comment by baozholu-AMD** [comment]
@Taldren76 We don't have a Sony STR-DH790 receiver, can you provide more useful information about this issue?

**Comment by Taldren76** [comment]
It seems it is any HDMI connection that doesn't have a video source causes it. From what I have heard from others, it happens with soundbars and other receivers as well.  SPDIF and component audio is obsolete and most high end audio is now using HDMI exclusively and for some reason, this disabled OpenGL on my (and others) AMD cards.

**Comment by Slaw6820** [comment]
@Taldren76 
Could you please provide more details on other users that are affected by similar issue? Maybe some links that point to discussion about similar issue?

**Comment by Taldren76** [comment]
Certainly: https://community.amd.com/t5/general-discussions/connecting-to-audio-receiver-with-hdmi-will-disable-opengl/td-p/725554

**Comment by baozholu-AMD** [comment]
@Taldren76 besides connect the Sony device, we also need to connect a monitor to the graphics crad (using DP or HDMI ?), right ?
is it only Opengl applications have issue? can you help verify DX/Vulkan apps ? 
you can download "GPUCapsViwer": it has d3d, ogl and vulkan demos
